crates/apps_lib/src/client/rpc.rs

```diff
@@ -359,6 +360,29 @@ pub async fn query_proposal_by_id<C: Client + Sync>(
     namada_sdk::rpc::query_proposal_by_id(client, proposal_id).await
 }
 
+/// Estimate MASP rewards for next MASP epoch
+pub async fn query_rewards_estimate(
+    context: &impl Namada,
+    args: args::QueryRewardsEstimate,
+) {
+    let mut shielded = context.shielded_mut().await;
+    let _ = shielded.load().await;
+    // Save the update state so that future fetches can be short-circuited
+    let _ = shielded.save().await;
```

Comment:
Do we really need t save immediately after load?

Reply:
Nope, bad copy pasta. Good catch